.code-samples.meilisearch.yaml

```diff
@@ -447,3 +479,30 @@ geosearch_guide_sort_usage_2: |-
   client.index('restaurants').search('', {
     sort: ['_geoPoint(48.8583701,2.2922926):asc', 'rating:desc'],
   })
+security_guide_search_key_1: |-
+  const client = new MeiliSearch({ host: "http://localhost:7700", "apiKey": apiKey })
+  client.index('patient_medical_records').search()
+security_guide_update_key_1: |-
+  const client = new MeiliSearch({ host: "http://localhost:7700", "apiKey": masterKey })
+  client.updateKey(d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4, {
```

Comment:
Actually all the places with the `d0552b41536279a0ad88bd595327b96f01176a60c2243e906c52ac02375f9bc4` we should add the quoting 👯‍♀️